tgui/packages/tgui/interfaces/Smelter.js

```diff
@@ -0,0 +1,103 @@
+import { toTitleCase } from 'common/string';
+import { useBackend, useLocalState } from '../backend';
+import { BlockQuote, Box, Button, NumberInput, Section, Table } from '../components';
```

Comment:
```suggestion
import { Box, Button, Section, Table } from '../components';
```
You're not using BlockQuote or NumberInput in this UI so it's not needed to import them.